lib/fork.js

```diff
@@ -39,6 +39,7 @@ module.exports = (file, opts, execArgv) => {
 		file,
 		baseDir: process.cwd(),
 		tty: process.stdout.isTTY ? {
+			colorDepth: process.stdout.getColorDepth ? process.stdout.getColorDepth() : 1,
```

Comment:
Perhaps leave as `undefined` if there is no `getColorDepth()`, and use that as a signal of whether to shim or not?

Or `null` if you prefer.